keyboards/qneatandtidy/dx65/hotswap/rev1/rules.mk

```diff
@@ -0,0 +1,5 @@
+# Do not put the microcontroller into power saving mode
+# when we get USB suspend event. We want it to keep updating
+# backlight effects.
```

Comment:
I'd say that enabling sleep when the usb suspends is probably a better behavior, over continuing animations